### ISSUE TITLE
[Bug Report] no data is shown just an error message 

### ISSUE THREAD

**Issue by johahauf** [SapuSeven/BetterUntis#245]
<details>
<summary>Logs</summary>

```
Anfragen-ID: 0
Fehlercode: 3
Fehlermeldung: o7.b: Field 'staffInfo' is required for type with serial name 'com.sapuseven.untis.models.untis.timetable.PeriodText', but it was missing
```
</details>

**Additional information**

- Android version: _10_
- BetterUntis version: _3.4.7 (31)_
- Installation source: _com.android.vending_

**Comment by Luis-L-B** [comment]
I had the same issue, it seems like units changed something again. Luckily (at least for me) there wasn't a lot of change required to fix it. 
@SapuSeven I was able to fix it by changing 
"val staffInfo: UnknownObject?" to 
"val staffInfo: UnknownObject? = null" 
in models/untis/timetable/PeriodText.kt and adding 
"val blockHash: UnknownObject? = null" 
in models/untis/timetable/Period. 
I hope this helps

**Comment by SapuSeven** [comment]
This has been fixed in version 3.4.8. You can opt-in as a beta tester on Google Play to receive the update sooner. Thank you @Luis-L-B for the suggestion.
